String/ValidateUrl.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @function ValidateURL
+ * @description validate the URL.
+ * @param {String} url - The input URL string
+ * @return {Boolean}
+ */
+const validateURL = (url) => {
+  const URL_PATTERN = /(https?:\/\/(?:www\.|(?!www))[^\s.]+\.[^\s]{2,}|www\.[^\s]+\.[^\s]{2,})/gi
```

Comment:
This regex will validate any strings which contain a valid link, for example:
`"link: https://www.google.com"`. Make sure the whole string matches the regex.

Reply:
yes, it is  `https://www.google.com` is a valid URL

Reply:
Fix that issue